Cargo.toml

```diff
@@ -20,7 +20,7 @@ hex = "0.4.3"
 rss = "2.0.3"
 atom_syndication = "0.12"
 ruma = { version = "0.8.2", features = ["events", "unstable-sanitize"] }
-reqwest = "0.11"
```

Comment:
I had to change that because the default features use native openssl bindings, which was harder to cross-compile

Reply:
I doubt we mind all that much :+1: